brave-sparkjava/src/main/java/com/github/kristofa/brave/sparkjava/BraveTracingSparkJava.java

```diff
@@ -0,0 +1,99 @@
+package com.github.kristofa.brave.sparkjava;
+
+import com.github.kristofa.brave.Brave;
+import com.github.kristofa.brave.ServerRequestInterceptor;
+import com.github.kristofa.brave.ServerResponseInterceptor;
+import com.github.kristofa.brave.ServerSpanThreadBinder;
+import com.github.kristofa.brave.ServerTracer;
+import com.github.kristofa.brave.http.DefaultSpanNameProvider;
+import com.github.kristofa.brave.http.HttpServerRequestAdapter;
+import com.github.kristofa.brave.http.HttpServerResponseAdapter;
+import com.github.kristofa.brave.http.SpanNameProvider;
+import com.github.kristofa.brave.servlet.internal.MaybeAddClientAddressFromRequest;
+import spark.ExceptionHandler;
+import spark.Filter;
+import spark.Request;
+import spark.Response;
+import zipkin.Constants;
+
+import static com.github.kristofa.brave.internal.Util.checkNotNull;
+
+public final class BraveTracingSparkJava {

```

Comment:
Yes,your advise is better.I will change BraveTracingSparkJava  to BraveTracing. As user add dependence ,they will know BraveTracing is used for traceing in SparkJava.